src/Indicator.vala

```diff
@@ -66,11 +74,29 @@ public class BluetoothIndicator.Indicator : Wingpanel.Indicator {
 
     public override void closed () {
     }
+
+    private void update_tooltip (bool state, bool paired) {
+        string description = _("is off");
```

Comment:
I think it's best (and easier for translators) to be less clever here, and just use the entire string.

```suggestion
        string description = _("Bluetooth is off");
```